rospix_classification/Cluster[] cluster_list
rospix_classification/ClusterCounts cluster_counts
